master/buildbot/status/web/console.py

```diff
@@ -284,24 +284,26 @@ def getAllBuildsForRevision(self, status, request, codebase, lastRevision,
             builder = status.getBuilder(builderName)
 
             # Make sure we are interested in this builder.
-            if categories and builder.category not in categories:
+            if tags and not builder.matchesAnyTag(tags):
                 continue
             if builders and builderName not in builders:
                 continue
 
             # We want to display this builder.
-            category = builder.category or "default"
+            tags = builder.getTags() or ["default"]
+            
             # Strip the category to keep only the text before the first |.
             # This is a hack to support the chromium usecase where they have
-            # multiple categories for each slave. We use only the first one.
+            # multiple tags for each slave. We use only the first one.
             # TODO(nsylvain): Create another way to specify "display category"
             #     in master.cfg.
-            category = category.split('|')[0]
-            if not builderList.get(category):
-                builderList[category] = []
+            if len(tags)==1:
+                tags = tags[0].split('|')
```

Comment:
I'm ok to remove it ... I dont fully understand why it's there. Looks like it was added in the first commit of Console (looks like 0.7.10) and hasnt been updated since. So, I would guess it's probably not used anymore? Plus, I think the "multiple tags" actually implements what this is trying to accomplish anyway. Shall I remove it?